api/stats.js

```diff
@@ -1,31 +1,73 @@
 import { config } from './../config.js';
 import { SnowflakeUtil, MessageEmbed } from "discord.js";
 
+const removeParentheses = word => word.replace(/\(|\)/g, '');
 export class Stats {
     constructor(discordClient) {
         this.client = discordClient;
         this.stats = {};
     }
 
-    initializeStats(prev_message) {
-        /*
-        This is a placeholder.
-        We have some limitations on how many messages we can load per request
-        we can have daily pulls, use the latest message as the last source of truth
-        and update with new stats
-        let stats = {
-            username1: {
-                messages: 5, 
-                words: 20, 
-                letters: 150
+    /*
+        Returns a snowflake which represents the time x days in the past
+        https://discord.com/developers/docs/reference#snowflakes
+    */
+    _generatDaySnowflake(days) {
+        let d = new Date();
+        d.setHours(0, 0, 0, 0);
+        d.setDate(d.getDate() + days);
+        return SnowflakeUtil.generate(d);
+    }
+
+    async _getStatsChannel() {
```

Comment:
I propose to declare in class functions using the arrow notation instead, that way it is clearer that "this" objects is not being overwritten.